### PR TITLE
OSDOCS-2875: global Azure availability sets in 4.10

### PR DESCRIPTION
**Pull request by jeana-redhat**
For [OSDOCS-2875](https://issues.redhat.com/browse/OSDOCS-2875), which documents [OCPCLOUD-1223](https://issues.redhat.com/browse/OCPCLOUD-1223)

Notes:
WIP, so far have attempted to refer to avail sets as an alternative to avail zones where appropriate. Added label to sample YAML.

Previews:
- [Sample YAML](https://deploy-preview-41316--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-azure.html#machineset-yaml-azure_creating-machineset-azure) in Creating a machine set on Azure
- [Sample YAML](https://deploy-preview-41316--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#machineset-yaml-azure_creating-infrastructure-machinesets) in Creating infrastructure machine sets
- [Machine API overview](https://deploy-preview-41316--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-azure.html#machine-api-overview_creating-machineset-azure)
- [Creating infrastructure machine sets for production environments](https://deploy-preview-41316--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#creating-infrastructure-machinesets-production) (Machine management book)
- [Creating infrastructure machine sets for production environments](https://deploy-preview-41316--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#post-install-creating-infrastructure-machinesets-production) (Post-install cluster tasks)
- [Short-circuiting machine health check remediation](https://deploy-preview-41316--osdocs.netlify.app/openshift-enterprise/latest/machine_management/deploying-machine-health-checks.html#machine-health-checks-short-circuiting_deploying-machine-health-checks)
- [Understanding pod affinity](https://deploy-preview-41316--osdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-pod-affinity.html#nodes-scheduler-pod-affinity-about_nodes-scheduler-pod-affinity)
- [MTC Known issues](https://deploy-preview-41316--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/troubleshooting-3-4.html#migration-known-issues_troubleshooting-3-4) - last one, BZ1947487. Not sure if this availability zones concern will also apply to availability sets.

### CODE REVIEW

File: modules/machine-health-checks-resource.adoc [hunk 1]
```diff
@@ -50,22 +50,19 @@ The `matchLabels` are examples only; you must map your machine groups based on y
 Short circuiting ensures that machine health checks remediate machines only when the cluster is healthy.
 Short-circuiting is configured through the `maxUnhealthy` field in the `MachineHealthCheck` resource.
 
-If the user defines a value for the `maxUnhealthy` field,
-before remediating any machines, the `MachineHealthCheck` compares the value of `maxUnhealthy`
-with the number of machines within its target pool that it has determined to be unhealthy.
-Remediation is not performed if the number of unhealthy machines exceeds the `maxUnhealthy` limit.
+If the user defines a value for the `maxUnhealthy` field, before remediating any machines, the `MachineHealthCheck` compares the value of `maxUnhealthy` with the number of machines within its target pool that it has determined to be unhealthy. Remediation is not performed if the number of unhealthy machines exceeds the `maxUnhealthy` limit.
```
Comment (jeana-redhat) [inline modules/machine-health-checks-resource.adoc:53]: No change, just unwrapped

**Comment by jeana-redhat** [comment]
@sunzhaohua2 I see you did the test plan for OCPCLOUD-868, so could you please take a look at this for QA verification?

**Review by skrthomas — APPROVED** [review]
LGTM!

**Pull request merged**